packages/client/package.json

```diff
@@ -102,6 +102,7 @@
     "karma-firefox-launcher": "^2.1.0",
     "karma-tap": "^4.2.0",
     "karma-typescript": "^5.5.1",
+    "karma-typescript-es6-transform": "^5.5.1",
```

Comment:
Is this introduction related to the quoted problem?

> client karma run is having some trouble with a downstream update: `Uncaught TypeError: TextDecoder is not a constructor`
> 
> it looks like it's from [`web-encoding v1.1.2`](https://github.com/Gozala/web-encoding/releases/tag/v1.1.2) which was just released, i'm looking into if it's an issue on our end or theirs.

Reply:
yes, so after I reported my bug they did a new release updating the browser file but it was an esmodules (es6) file so I needed this plugin to transform it to es5 to work in our pipeline.